src/shapes/object.class.ts

```diff
@@ -1,2049 +1,2136 @@
-//@ts-nocheck
+// @ts-nocheck
+import type { TClassProperties, TDegree, TSize, TFiller } from '../typedefs';
+import { fabric } from '../../HEADER';
 import { cache } from '../cache';
 import { config } from '../config';
 import { VERSION } from '../constants';
+import { CommonMethods } from '../mixins/shared_methods.mixin';
 import { Point } from '../point.class';
 import { capValue } from '../util/misc/capValue';
 import { pick } from '../util/misc/pick';
 import { runningAnimations } from '../util/animation_registry';
 import { enlivenObjectEnlivables } from '../util/misc/objectEnlive';
+import { clone } from '../util/lang_object';
+import { toFixed } from '../util/misc/toFixed';
+import { capitalize } from '../util/lang_string';
+import { degreesToRadians } from '../util/misc/radiansDegreesConversion';
+import { createCanvasElement } from '../util/misc/dom';
+
+type StaticCanvas = any;
+type Canvas = any;
```

Comment:
Use the Canvas type from `__types__`?

Reply:
as soon as Canvas is converted, we can trash those all away